src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/Resources/ViewsTestMessages.resx

```diff
@@ -0,0 +1,126 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
we could create one .resx file for all resources in all views - so maybe we should consider renaming it to `ViewsMessages`?